src/utils/config/project.py

```diff
@@ -42,6 +42,20 @@ def __init__(
         self._version_manager = version_manager
         self._config_dict: Optional[Dict[str, Any]] = None
 
+    @property
+    def repo_path(self) -> Optional[Path]:
```

Comment:
It seems, Repo supports `search_parent_directories` flag, we don't need our own implementation